sql/pgwire/v3.go

```diff
@@ -324,7 +317,11 @@ func (c *v3Conn) sendResponse(resp driver.Response, formatCodes []formatCode) er
 				c.writeBuf.initMsg(serverMsgDataRow)
 				c.writeBuf.putInt16(int16(len(row.Values)))
 				for i, col := range row.Values {
-					switch formatCode := formatCodes[i]; formatCode {
+					fmtCode := formatText
+					if formatCodes != nil {
+						fmtCode = formatCodes[i]
```

Comment:
The `formatCodes` list may contain a single value, in which case that value is applied to all parameters (not sure when anyone would do that, but it's in the protocol spec).